batch/batch/front_end/front_end.py

```diff
@@ -2139,12 +2187,21 @@ async def ui_get_job(request, userdata, batch_id):
 
     # Not all logs will be proper utf-8 but we attempt to show them as
     # str or else Jinja will present them surrounded by b''
-    job_log_strings_or_bytes = {}
-    for container, log in job_log_bytes.items():
-        try:
-            job_log_strings_or_bytes[container] = log.decode('utf-8') if log is not None else None
-        except UnicodeDecodeError:
-            job_log_strings_or_bytes[container] = log
+    job_log_strings_or_bytes: Dict[str, Union[str, bytes, None]] = {}
+    possibly_truncated_logs = set()
+    for container, log_stream in job_log_streams.items():
```

Comment:
Same argument here. We're not loading the streams in parallel anymore so stuffing them all in that gather above is less compelling.

Reply:
Or also see above